### ISSUE TITLE
Project.getRepo is deprecated.

### ISSUE THREAD

**Issue by ivajo26** [jacogr/atom-git-control#82]
Use ::getRepositories instead
```
Project.getRepo (/usr/share/atom/resources/app.asar/src/project.js:579:12)
Object.<anonymous> (/home/ivajo26/.atom/packages/git-control/lib/git.coffee:15:17)
Object.<anonymous> (/home/ivajo26/.atom/packages/git-control/lib/git.coffee:289:4)
Module._compile (module.js:452:26)
Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
Module.load (module.js:347:32)
```

**Comment by nounoursheureux** [comment]
It should have been fixed by 008fe7491bdeaaaf85bee4c17f8c08087ba8d68c , please update the package and if the issue persists tell us :+1:

**Comment by MarcelMue** [comment]
Closed for now, feel free to reopen if the deprecation still exists somehow.
